apps/parking_lot/tests/test_apis.py

```diff
@@ -113,3 +114,49 @@ def test_post_parked_item(self):
         self.authenticate()
         response = self.client.post(self.url, data)
         assert response.status_code == 201
+
+    def test_post_multiple_parked_item(self):
+        old_count = ParkedItem.objects.count()
+        self.country1 = CountryFactory.create()
+        self.country2 = CountryFactory.create()
+        self.assigned_to1 = create_user_with_role(USER_ROLE.MONITORING_EXPERT_REVIEWER.name)
+        self.assigned_to2 = create_user_with_role(USER_ROLE.MONITORING_EXPERT_REVIEWER.name)
+        data = [
+            {
+                "title": "test_parking",
+                "url": "http://google.com",
+                "country": self.country1.id,
```

Comment:
```suggestion
                "country_iso3": self.country1.iso3,
```
Same for next object